torpaste.py

```diff
@@ -1,4 +1,4 @@
-#!bin/python
```

Comment:
This particular line is used locally to run TorPaste in a `virtualenv`. If use use Docker or a Production Web Server (which you should), then this line is not used anywhere. Therefore that's a "hack" I use to allow people who don't do development on Docker and instead prefer `virtualenv` to run the project in Development mode.